bundles/org.eclipse.e4.ui.css.swt/src/org/eclipse/e4/ui/css/swt/properties/preference/EclipsePreferencesHandler.java

```diff
@@ -61,5 +65,18 @@ protected void overrideProperty(IEclipsePreferences preferences, String name, St
 			preferences.put(name, value);
 			EclipsePreferencesHelper.appendOverriddenPropertyName(preferences, name);
 		}
+		ColorRegistry colorRegistry = JFaceResources.getColorRegistry();
+		if (colorRegistry != null) {
+			RGB rgb = colorRegistry.getRGB(name);
+			if (rgb != null) {
+				IEclipsePreferences defaultPrefs = DefaultScope.INSTANCE.getNode(preferences.name());
+				if (defaultPrefs != null) {
+					String existing = defaultPrefs.get(name, null);
+					if (existing != null && value != null && existing.equals(value) == false) {
```

Comment:
can the check "value == null" be put at the beginning of the method? But in line 65 "null" is set to the preference without checking it. So maybe we can at least move it before line 68.